### ISSUE TITLE
Login/World Server UDPListener just...dies

### ISSUE THREAD

**Issue by pschord** [psforever/PSF-LoginServer#446]
While playing, the login server's UDP port just disappeared, preventing new logins. Right before the last sessions that made it, there was a spike in processing time, which could be from DB backlog, or something else entirely.

![image](https://user-images.githubusercontent.com/14133344/82153930-7e227780-9838-11ea-8d0d-6ee66f61ae43.png)

The weirdest part was that the UDP port for login (51000) was completely missing from the Kernel's UDP port output:

```
$ ss -ul | grep 51
UNCONN     0      0      ::ffff:62.210.250.199:51000  :::* # this was missing
UNCONN     0      0      ::ffff:62.210.250.199:51001  :::*
```

This could indicate that the session-router itself died or that the UdpListener did - or something else entirely.

**Comment by pschord** [comment]
Same thing just happened to WorldServer during a large play test! No idea what causes it

**Comment by pschord** [comment]
Could be related to
https://github.com/akka/akka/issues/22076
https://github.com/akka/akka/issues/17748
Fix is probably upgrading Akka + Scala to 2.12

**Comment by NickPSF** [comment]
Probably fixed?
